code/modules/admin/panels/player_panel.dm

```diff
@@ -464,20 +464,24 @@
 
 
 	if(!isnewplayer(M))
-		body += {"<br><br>
-			<b>Other actions:</b>
-			<br>
-			<a href='?src=[ref];thunderdome=[REF(M)]'>Thunderdome</a> |
-			<a href='?src=[ref];gib=[REF(M)]'>Gib</a>"}
+		body += "<br><br>"
```

Comment:
```suggestion
		body += "<br><br>\
			<b>Other actions:</b>\
			<br>\
			<a href='?src=[ref];thunderdome=[REF(M)]'>Thunderdome</a> | \
			<a href='?src=[ref];gib=[REF(M)]'>Gib</a>"
```
A single operation while retaining clarity.

Reply:
I'd argue this is much easier on the eyes, plus admin code doesn't have to squeeze every last drop of performance.